optimization/+DoseConstraints/matRad_ObjectiveConstraint.m

```diff
@@ -0,0 +1,111 @@
+classdef matRad_ObjectiveConstraint < DoseConstraints.matRad_DoseConstraint
```

Comment:
I find the name a little ambiguous. Could we maybe switch to matRad_DoseConstraintFromObjective?